CHANGELOG.md

```diff
@@ -1,9 +1,5 @@
-# 7.5.2
-- Set the class of a swizzled object back to its original class upon deallocation. (#8321)
```

Comment:
There still should be an update for the previous fix.

Reply:
Yep I'm opening a follow-up PR with previous fix and previous fix's CHANGELOG entry

Reply:
Order:
- Merge this revert into `main`
- Merge #57 (previous fix) into `main` 
- Retag `CocoaPods-8.5.1` to `main`'s latest commit